rsocket-core/src/main/java/io/rsocket/RSocketRequester.java

```diff
@@ -205,7 +205,7 @@ public boolean isDisposed() {
       return Mono.error(err);
     }
 
-    final int streamId = streamIdSupplier.nextStreamId();
+    final int streamId = streamIdSupplier.nextStreamId(receivers);
```

Comment:
why not make it a long?

Reply:
According to the protocol the stream id is an unsigned 32-bit number